ui/imports/Utils.qml

```diff
@@ -567,7 +567,7 @@ QtObject {
                     return [false, qsTrId("you-need-to-repeat-your-password")];
                 } else if (repeatPasswordField.text !== firstPasswordField.text) {
                     //% "Passwords don't match"
```

Comment:
Maybe you need to remove these auto-generated comments as well?
I believe script inserts them when changing qsTr to qsTrId.

Reply:
You're right.

I think in this case, the UI was not displaying the right string because either `lrelease` was not run or Utils.qml is not part of the `SOURCE` list in the `.pro` file as per https://hackmd.io/@status-desktop/B1naRjxh_/https%3A%2F%2Fhackmd.io%2F%40status-desktop%2FB1eOaf-nd.
I think it's the latter.

So to "permanently" fix this issue, you just need to add `imports/` to the `SOURCE` list of the `.pro` file if it isn't there.